resolving/npm-resolver/src/index.ts

```diff
@@ -356,6 +371,7 @@ async function resolveNpm (
     resolvedVia: 'npm-registry',
     publishedAt: meta.time?.[pickedPackage.version],
     normalizedBareSpecifier,
+    provenanceDowngraded,
```

Comment:
why do we need to return this boolean instead of just throwing an error from isProvenanceDowngraded?